install/kubernetes/helm/istio/values.yaml

```diff
@@ -224,6 +224,14 @@ global:
     # This controls the 'policy' in the sidecar injector.
     autoInject: enabled
 
+    # You can use the field called alwaysInjectSelector and neverInjectSelector which will always inject the sidecar or
+    # always skip the injection on pods that match that label selector, regardless of the global policy.
+    # See https://istio.io/docs/setup/kubernetes/additional-setup/sidecar-injection/#more-control-adding-exceptions
+
+    neverInjectSelector: []
```

Comment:
I don't thin it's a good idea to put in under `proxy` section.

Reply:
Just one line above `.Values.global.proxy.autoInject` is used in the same ConfigMap. That's why I thought here is a good candidate for this setting also. What's a better place?

Reply:
I would place it under 'sidecarInjectorWebhook' as the other contents of the ConfigMap are also there.

Reply:
@ntausch Under .Values.sidecarInjectorWebhook there is only enable/disable flag:
```
sidecarInjectorWebhook:
  enabled: true
```

Am I looking at somewhere wrong?

Second point is, although there is a dedicated sidecarInjectorWebhook subchart, the ConfigMap is created in the parent chart. [See here](https://github.com/istio/istio/blob/master/install/kubernetes/helm/istio/templates/sidecar-injector-configmap.yaml). Maybe this should be moved to subchart? What do you think?

Reply:
Under `.Values.sidecarInjectorWebhook` are the further contents of the ConfigMap 'istio-sidecar-injector', but also some settings for the mutating webhook configuration of 'istio-sidecar-injector'. 

The ConfigMap is defined in the file 'install/kubernetes/helm/istio/templates/sidecar-injector-configmap.yaml'

Examples:

- `.Values.sidecarInjectorWebhook.rewriteAppHTTPProbe`
- `.Values.sidecarInjectorWebhook.enableNamespacesByDefault`